apps/server/src/services/institutionSyncer.ts

```diff
@@ -0,0 +1,77 @@
+import { setIntervalAsync } from 'set-interval-async'
+import config from '../config'
+import { info, warning as logWarning } from '../infra/logger'
+import {
+  deleteRemovedInstitutions,
+  updateInstitutions
+} from '../services/ElasticSearchClient'
+import type { CachedInstitution } from '../shared/contract'
+import { getAccessToken } from './auth0Service'
+import { INSTITUTION_ETAG_REDIS_KEY } from './storageClient/constants'
+import { get, setNoExpiration } from './storageClient/redis'
+
+const RESPONSE_NOT_MODIFIED = 304
+const SUCCESS_RESPONSE = 200
+const UNAUTHORIZED_RESPONSE = 401
+
+export async function setInstitutionSyncSchedule(minutes: number = 1) {
+  return setIntervalAsync(
+    async () => {
+      info('Checking for institution list updates')
+      await loadCachedInstitutions()
+    },
+    minutes * 60 * 1000
+  )
+}
+
+export const loadCachedInstitutions = async () => {
```

Comment:
loadCachedInstitutions is also confusing to me.

updateInstitution? syncInstitutions?